.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -20,17 +20,21 @@ need to be checked.
 - [ ] I added unit tests
 - [ ] I ran all tests locally and everything went fine
 - [ ] affected documentation is fixed
-- [ ] I added code comments, logging, and assertions (see doc/CODING.md)
-- [ ] meta data is updated (e.g. README.md of plugins and doc/METADATA.md)
+- [ ] I added code comments, logging, and assertions (see [Coding Guidelines](/doc/CODING.md))
```

Comment:
Is there an advantage of adding a slash here? I ask, because adding slashes breaks the ability of my editor to open a link via <kbd>⌘</kbd> + <kbd>⌤</kbd>, since it assumes that `/` specifies the root folder of my hard disk.

Reply:
It is needed to make this link work (absolute path).

The alternative would be ../doc/ which is longer and less stable (needs to be changed if either of these two documents are moved).